op-service/testutils/mockrpc/matchers.go

```diff
@@ -0,0 +1,71 @@
+package mockrpc
```

Comment:
This was pulled in from the SR to allow me to make a mock RPC server for testing purposes. Once this is merged we can update the SR to use this lib.